submissions/chabuChabu/javaScript/signup.js

```diff
@@ -0,0 +1,121 @@
+let submitForm = document.getElementById('validateSignUp'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    emailAddress = document.getElementById('emailAddress'),
+    password = document.getElementById('password'),
+    confirmPassword = document.getElementById('confirmPassword')
+;
+
+function required(field, event) {
+    if(field.value === '') {
+        field.nextElementSibling.innerHTML = "This is a required field";
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function fNameValidation(event){
+    let fNameRegex = /^[A-Za-z]+$/;
+    if(!firstName.value.match(fNameRegex)){
+        firstName.nextElementSibling.innerHTML = "Please enter your first name";
+        event.preventDefault();
+        return false;
+    }else{
+        firstName.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function lNameValidation (event){
+    let lNameRegex = /^[A-Za-z]+$/;
+    if(!lastName.value.match(lNameRegex)){
+        lastName.nextElementSibling.innerHTML = "Please enter your last name";
+        event.preventDefault();
+        return false;
+    }else{
+        lastName.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function phoneValidation(event){
+    let phoneRegex = /^[+]?[0-9]{8,14}$/i;
+    if(phoneNumber.value === ""){
+        phoneNumber.nextElementSibling.innerHTML = "Please enter the phone number";
+        event.preventDefault();
+        return false;
+    }else if (!phoneNumber.value.match(phoneRegex)){
+        phoneNumber.nextElementSibling.innerHTML = "Please enter a valid number";
+        event.preventDefault();
+    }else{
+        phoneNumber.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function emailValidation(event){
+    let emailRegex = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
+    if(emailAddress.value === ""){
```

Comment:
The `required` function you created already handle this check. kindly reuse the function